packages/integration-test-app/windows/integrationtest/integrationtest.vcxproj

```diff
@@ -178,6 +180,11 @@
       <SubType>Designer</SubType>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.210708.4" />
+    <PackageReference Include="ReactNative.Hermes.Windows" Version="0.8.0-ms.0" />
+    <PackageReference Include="$(WinUIPackageName)" Version="$(WinUIPackageVersion)" />
```

Comment:
Consider adding this to "external property sheets", so that this is controlled by the npm package and it automatically updated when the user updates their version of RNW, instead of needing to update their package manually.

Reply:
This is a good idea, but we should add this on a follow-up change.

Note, this is "standard" way to handle dependencies even in modern .NET projects.
It happens to replace the include/validate churn from using `packages.config`.